.mailmap

```diff
@@ -0,0 +1,21 @@
+Andreas Grüninger <andreas.grueninger@lgl.bwl.de> <andreas.grueninger@server>
+Andrew Beekhof <andrew@beekhof.net> <abeekhof@nsew.lab.bos.redhat.com>
+Bin Liu <bliu@suse.com>
+Bin Liu <bliu@suse.com> <liu4480@126.com>
+David Vossel <davidvossel@gmail.com> <dvossel@dvossel-work.awesome>
+David Vossel <davidvossel@gmail.com> <dvossel@gmail.com>
+David Vossel <davidvossel@gmail.com> <dvossel@redhat.com>
+David Vossel <davidvossel@gmail.com> <root@dvossel-work.awesome>
+Hideo Yamauchi <renayama19661014@ybb.ne.jp>
+Hideo Yamauchi <renayama19661014@ybb.ne.jp> <guest@example.com>
+Hideo Yamauchi <renayama19661014@ybb.ne.jp> <yamauchi@.intellilink.co.jp>
+Hu Fu <askfuhu@gmail.com>
+Jan Pokorný <jpokorny@redhat.com>
+Kazunori INOUE <kazunori.inoue3@gmail.com> <root@x3650f.(none)>
+Lars Ellenberg <lars.ellenberg@linbit.com> <makepkg@buildsystem.linbit>
+Lidong Zhong <lzhong@suse.com> <ultra.gammaray@gmail.com>
+Yan Gao <ygao@suse.com>
+Yan Gao <ygao@suse.com> <ygao@novell.com>
+Yuusuke Iida <yusk.iida@gmail.com>
+Yuusuke Iida <yusk.iida@gmail.com> <iidayuus@intellilink.co.jp>
+Yuusuke Iida <yusk.iida@gmail.com> <yusuke.iida@gmail.com>
```

Comment:
There's no way any reviewer (including me) can be expected to verify every commit's author configuration or check the mailmap file for potential conflicts with every commit. Those two entries mentioned do not contain enough information to identify a particular individual, so they should not be mapped to a particular individual's authorship.